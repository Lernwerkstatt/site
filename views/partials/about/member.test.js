const teamMember = require("./member.hbs");

describe("member.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      teamMember({
        name: "Max Muster",
        id: "max",
        img: "img/logo.png",
        description: "Lorem ipsum."
      })
    ).toMatchSnapshot();
  });
});
