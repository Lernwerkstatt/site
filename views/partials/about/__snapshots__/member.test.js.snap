// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member.hbs should match the snapshot 1`] = `
"<div class=\\"d-inline-block mr-3 pb-3\\" data-toggle=\\"collapse\\" data-target=\\"#max\\">
  <img src=img/logo.png alt=Max Muster class=\\"img-fluid rounded-circle\\" style=\\"height: 160px; width: 160px;\\">
  <p class=\\"lead\\">Max Muster</p>
</div>
<div id=\\"max\\" class=\\"collapse\\" data-parent=\\"#team\\">
  <div class=\\"card-body mt-n5 mb-2\\">
    Lorem ipsum.
  </div>
</div>
"
`;
